alacritty/src/display.rs

```diff
@@ -431,6 +467,7 @@ impl Display {
 
         // Resize renderer.
         let physical = PhysicalSize::new(self.size_info.width as u32, self.size_info.height as u32);
+        self.window.set_min_inner_size(Some(min_size));
```

Comment:
We shouldn't always do this. Constantly setting the min size of the window while resizing seems extremely wasteful. We only need to do this when the font size changed. Technically the padding might also change this, but that's not really handled here.

Reply:
Okay, so I'll only set the min size when the font is updated.